queries/leo/highlights.scm

```diff
@@ -115,7 +113,15 @@
 (boolean_literal) @boolean
 
 (constant_declaration
-  (identifier) @constant)
+  (identifier
+    (constant_identifier) @constant))
+
+(variable
+  (constant_identifier) @constant)
```

Comment:
Question: Can this be reduced to
```suggestion
(constant_identifier) @constant
```

Reply:
No, see below.

Reply:
@lucario387 This was my original plan, but unfortunately for me, records, and structs both have in their struct_component_declarations constant_identifiers, but I dont want them to LOOK like constants.
If I accept your change this will override struct_component_declarations. If you check the comments of @ribru17 above, he had issues with exactly this. 
So I would like to keep things as they are.